ic-os/components/setupos-scripts/devices.sh

```diff
@@ -27,17 +30,25 @@ function copy_config_files() {
     fi
```

Comment:
I haven't removed the logic to copy config.ini and deployment.json to the hostos config partition, so the current changes should have no effect on HostOS yet.